Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.cpp

```diff
@@ -913,6 +915,20 @@ int CLuaVehicleDefs::GetVehicleWheelStates ( lua_State* luaVM )
     return 1;
 }
 
+int CLuaVehicleDefs::IsVehicleWheelCollided ( lua_State* luaVM )
+{
+    CClientVehicle* pVehicle = NULL; eWheels wheel;
+    CScriptArgReader argStream ( luaVM );
+    argStream.ReadUserData ( pVehicle );
+    argStream.ReadEnumStringOrNumber ( wheel );
+
+    if ( !argStream.HasErrors () )
+        lua_pushboolean ( luaVM, pVehicle->IsWheelCollided ( wheel ) );
+    else
+        lua_pushboolean ( luaVM, false );
```

Comment:
`m_pScriptDebugging->LogCustom ( luaVM, argStream.GetFullErrorMessage() );` is missing here